en/features/details/transaction-withholding.md

```diff
@@ -0,0 +1,45 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+layout: post
+lang: en
+columns: 1
+id: details-transaction-withholding
+title: Details about transaction withholding
+breadcrumbs:
+  - features
+  - verification
+---
+> Mallory offers Alice $1,000 USD in exchange for some bitcoins.  Alice
+> sends the bitcoins but the transaction never seems to confirm and so
+> Alice is unable to insist that Mallory pay the cash.  It turns out the
+> transaction did confirm, so Alice gave away her bitcoins for nothing.
+
+- **Wallet service**{:.fgred} users only see the transactions the service
+  choose to show them.
+
+- **Light wallets**{:.fgred} users only see the transactions their full
+  node peers choose to send them, even if those transactions were
+  included in a block the light wallet knows about.
+
+- **Bitcoin Core**{:.fggreen} users see all transactions included in
+  received blocks. If Bitcoin Core hasn't received a block for too long,
+  it displays a catching-up progress bar in the graphical user
+  interface or a warning message in the CLI/API user interface.
+
+<div class="callout" markdown="block">
+Unless you use Bitcoin Core, you can never be sure that your bitcoin balance
+is correct according to the block chain.
```

Comment:
```suggestion
is correct or that the bitcoins your receive are real bitcoin.
```

Reply:
We cover the validity of bitcoins in the chain hijacking section/page, so I don't think we need to double down on it here.  I'll drop the "according to the block chain" part.

Reply:
:+1: